### PR TITLE
Nand6028/callout fixes

### PR DESCRIPTION
**Pull request by nandinirao**
Assigned to @michael-tims 
@ldanzinger Could you also please review.

Major changes:
 - Adding leaderPositionEnum var to make using the Enum easier without having to import the js file.
 - Adding accessoryButton and accessoryButtonHidden bool.
 - Fix alpha, cornerRadius defaults and fix case on a few properties
 - Reduce font and callout sizes

### CODE REVIEW

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml [hunk 1]
```diff
@@ -164,9 +198,9 @@ Item {
     /*! \internal */
     property real calloutMaxWidth: 210
     /*! \internal */
-    property real calloutMaxHeight: 100
+    property real calloutMaxHeight: 50
     /*! \internal */
-    property real calloutMinWidth: calloutMaxWidth
+    property real calloutMinWidth: 95
```
Comment (michael-tims) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml]: Should these width and height values be scale factored?

**Comment by michael-tims** [comment]
Done reviewing

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml [hunk 2]
```diff
@@ -43,26 +66,11 @@ Item {
...
-            \li Enums.LeaderPosition.Automatic (8)
-        \endlist
+        \brief The property to set the leader position of the callout.
 
-        Automatic will decide the best placement, based on the
-        location of the callout within the MapView.
+        For example, to set the leader line to the top of the callout use:
 
-        The default is \c Enums.LeaderPosition.Bottom.
+        Callout.leaderPosition: leaderPositionEnum.Top
```
Comment (ldanzinger) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml]: add `\c` at the beginning of this line, so it shows up as code in the doc

**Comment by ldanzinger** [comment]
looks good, just the one doc suggestion

File: Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml (hunk 1 above)
Comment (nandinirao) [inline Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml]: No we do it later here: https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/v.next/Imports/ArcGIS/Runtime/Toolkit/Controls.2.0/Callout.qml#L272

**Comment by michael-tims** [comment]
Looks like just the one doc suggestion is needed and then these PRs can be merged.

**Comment by nandinirao** [comment]
@ldanzinger I updated the doc.

**Comment by nandinirao** [comment]
@michael-tims This is ready to merge.

**Pull request merged**